connectivity/FEATURE_BLE/source/cordio/source/GattServerImpl.cpp

```diff
@@ -647,10 +685,13 @@ ble_error_t GattServer::read(
     uint16_t *buffer_length
 )
 {
+    tr_info("Read attribute %d", att_handle);
```

Comment:
maybe print this later when the result is known then you can print the value
(in which case the error message shouldn't be context sensitive and should explain: "Attribute read failed - not found in server")

Reply:
where is it logged?
please only resolve conversation when you fix the problem or reply to the comment explaining why the suggestion is wrong

Reply:
Read and write should probably be `tr_debug` traces as these functions may be called quite often. Please add the buffer value into the trace. 

To do that, you can move the trace right beffore the return and add some context to the `tr_error`.